modules/generators/config_files/filezilla_config/secgen_metadata.xml

```diff
@@ -7,46 +7,14 @@
   <name>Filezilla Config File Generator</name>
   <author>Jack Biggs</author>
   <module_license>MIT</module_license>
-  <description>Creates a Filezilla config file which includes either 1 or 2 sets of plaintext credentials</description>
+  <description>Creates a Filezilla config file which includes a host, a port and a password</description>
 
   <type>string_generator</type>
   <type>local_calculation</type>
   <platform>linux</platform>
   
-  <read_fact>ftp_user</read_fact>
-  <read_fact>ftp_pass</read_fact>
-  <read_fact>ftp_user</read_fact>
-  <read_fact>ftp_pass</read_fact>
+  <read_fact>host</read_fact>
+  <read_fact>port</read_fact>
+  <read_fact>password</read_fact>
```

Comment:
Please add default inputs for these. I'll merge, based on our discussions today, but please put in a new PR just to add the default inputs for these